src/corefx/System.Globalization.Native/casing.cpp

```diff
@@ -0,0 +1,65 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+#include <assert.h>
+#include <stdint.h>
+#include <unicode/uchar.h>
+#include <unicode/utf16.h>
+
+/*
+Function:
+ChangeCase
+
+Preforms upper or lower casing of a string into a new buffer, preforming special casing for turkish, if needed.
+*/
+extern "C" void ChangeCase(const UChar* lpSrc, int32_t cwSrcLength, UChar* lpDst, int32_t cwDstLength, int32_t bToUpper, int32_t bTurkishCasing)
```

Comment:
Nit, but my preference would be to avoid the Hungarian notation in our CoreFX PAL API.